packages/password-field/src/vaadin-password-field.js

```diff
@@ -257,20 +263,13 @@ export class PasswordField extends TextField {
   }
 
   /** @private */
-  _onRevealButtonTouchend(e) {
-    // Cancel the following click event
+  _onRevealButtonMouseDown(e) {
+    // Cancel the following focusout event
     e.preventDefault();
-    this._togglePasswordVisibility();
 
     // By preventing `focusout` event, we also suppress related `change` event.
-    // Detect if that was the case and if so, dispatch `change` event manually.
-    if (this.__lastChange !== this.inputElement.value) {
-      this.__lastChange = this.inputElement.value;
-
-      this.validate();
-
-      this.dispatchEvent(new CustomEvent('change', { bubbles: true }));
-    }
+    // Set the flag to dispatch `change` event manually when field loses focus.
+    this.__pendingChange = true;
```

Comment:
Looks like it's unused.

Reply:
Thanks, removed. This was a leftover from another approach I tried.